snow/engine/snowman/transitive_test.go

```diff
@@ -2540,42 +2146,17 @@ func TestEngineBubbleVotesThroughInvalidBlock(t *testing.T) {
 func TestEngineBubbleVotesThroughInvalidChain(t *testing.T) {
 	require := require.New(t)
 
-	vdr, _, sender, vm, te, gBlk := setupDefaultConfig(t)
+	vdr, _, sender, vm, te := setupDefaultConfig(t)
 	expectedVdrSet := set.Of(vdr)
 
-	// [blk1] is a child of [gBlk] and currently passes verification
-	blk1 := &snowman.TestBlock{
-		TestDecidable: choices.TestDecidable{
-			IDV:     ids.GenerateTestID(),
-			StatusV: choices.Processing,
-		},
-		ParentV: gBlk.ID(),
-		HeightV: 1,
-		BytesV:  []byte{1},
-	}
-	// [blk2] is a child of [blk1] and cannot pass verification until [blk1]
-	// has been marked as accepted.
-	blk2 := &snowman.TestBlock{
-		TestDecidable: choices.TestDecidable{
-			IDV:     ids.GenerateTestID(),
-			StatusV: choices.Processing,
-		},
-		ParentV: blk1.ID(),
-		HeightV: 2,
-		BytesV:  []byte{2},
-		VerifyV: errInvalid,
-	}
-	// [blk3] is a child of [blk2] and will not attempt to be issued until
-	// [blk2] has successfully been verified.
-	blk3 := &snowman.TestBlock{
-		TestDecidable: choices.TestDecidable{
-			IDV:     ids.GenerateTestID(),
-			StatusV: choices.Processing,
-		},
-		ParentV: blk2.ID(),
-		HeightV: 3,
-		BytesV:  []byte{3},
-	}
+	blks := BuildChain(Genesis, 3)
+	blk1 := blks[0]
```

Comment:
we're changing the blk1 status here (it was Processing but here it is genesis, which is Accepted. Does this matter?